src/N3Parser.js

```diff
@@ -1035,14 +1047,36 @@ export default class N3Parser {
       return quads;
     }
 
-    // Parse asynchronously otherwise, executing the read callback when a token arrives
-    this._callback = quadCallback;
-    this._lexer.tokenize(input, (error, token) => {
+    let processNextToken = (error, token) => {
       if (error !== null)
         this._callback(error), this._callback = noop;
-      else if (this._readCallback)
+      else if (this._readCallback) {
         this._readCallback = this._readCallback(token);
-    });
+      }
```

Comment:
Let's keep the `{…}` off for single lines (the linter should normally complain, weird).

Reply:
Fixed